blivet/dbus/constants.py

```diff
@@ -19,7 +19,7 @@
 #
 
 BUS_NAME = "com.redhat.Blivet1"
-BASE_OBJECT_PATH = "/com/redhat/Blivet1"
+BASE_OBJECT_PATH = "/com/redhat/Blivet0"
```

Comment:
I think we need a documentation for the DBus API and mention that it is unstable there. Also there's the ``release_notes.rst`` file which should mention this.